modules/subscriptions/views.php

```diff
@@ -358,8 +358,17 @@ static function render_widget_subscription_form( $args, $instance, $subscribe_em
 					$email_field_id = 'subscribe-field' . self::$instance_count > 1
 						? '-' . self::$instance_count
 						: '';
+					$label_field_id = $email_field_id . '-label';
```

Comment:
Is `-label` ok for an ID, when `$email_field_id` is an empty string?

Reply:
Thanks for reviewing this, Jeremy!

I believe, given the code above:
```
$email_field_id = 'subscribe-field' . self::$instance_count > 1 ? '-' . self::$instance_count : '';
```
the `$email_field_id` will not be empty. 

If there's only one occurrence of the subscription form this will be 

`$email_field_id = 'subscribe-field'`;
`$label_field_id = 'subscribe-field-label';`

Reply:
I think that if `self::$instance_count` is equal to `1`, `$email_field_id` will be an empty string:
https://github.com/Automattic/jetpack/blob/c95e5fdb82110b54cb5e18b89b1558e37caead50/modules/subscriptions/views.php#L360

Thus causing `$label_field_id` to be set to `-label`

Reply:
Yes, you are right! 👍  
Seems that was an error, as the original author probably didn't mean for the `$email_field_id` to actually ever be empty.
Fixed that, nice catch!